apps/src/templates/sectionProgress/standards/StandardDescriptionCell.jsx

```diff
@@ -31,11 +38,36 @@ class StandardDescriptionCell extends Component {
     if (this.props.lessonsForStandardStatus) {
       return this.props.lessonsForStandardStatus.map((lesson, index) => {
         return (
-          <ProgressBoxForLessonNumber
-            key={lesson.lessonNumber}
-            completed={lesson.completed}
-            lessonNumber={lesson.lessonNumber}
-          />
+          <span>
+            <ReactTooltip
```

Comment:
Could we move the ReactTooltip inside ProgressBoxForLessonNumber?

Reply:
Sounds good! I moved the tooltips and added lesson to a new standards shapes file.

Reply:
Wait - I don't think we actually want the tooltip to live inside the ProgressBoxForLessonNumber component because we'll use that component in the dialogs where we don't want it to have a tooltip. 
<img width="486" alt="Screen Shot 2020-01-09 at 1 09 50 PM" src="https://user-images.githubusercontent.com/12300669/72105040-93632480-32e1-11ea-94ed-7b08b6fd54fd.png">

Reply:
Ya makes sense we wouldn't want them there. Good catch!